File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        LinkedHashSet<Integer> set = new LinkedHashSet<Integer>();
        for(int x : nums) set.add(x);
        int i = 0;
        for(int ele : set) {
            nums[i]  = ele;
            i++;
        }
        return set.size();
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        j=1
        curr=nums[0]
        for i in range(1,len(nums)):
            if nums[i]!=curr: 
                curr=nums[i]
                temp=nums[i]
                nums[i]=nums[j]
                nums[j]=temp
                j+=1
                
        return j
        